src/mount.ts

```diff
@@ -47,6 +47,7 @@ interface MountingOptions<Props, Data = {}> {
     ? Partial<Data>
     : never
   props?: Props
+  propsData?: Props
```

Comment:
Looks good, what do you think about marking this as deprecated?

https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#deprecated-support